helmfile.d/0450.klyshko-config.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2023 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/carbynestack.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# Deploys configuration for the Carbyne Stack Klyshko Offline Tuple Generation service.
+#
+# References:
+#     https://github.com/carbynestack/klyshko
+#
+releases:
+  - name: {{ requiredEnv "RELEASE_NAME" }}-klyshko-config
+    chart: ../charts/klyshko
+    values:
+      - scheduler:
+          enabled: {{ env "IS_MASTER" | default "false" }}
+          concurrency: {{ env "KLYSHKO_CONCURRENCY" | default 2 }}
+          threshold: {{ env "KLYSHKO_THRESHOLD" | default 50000 }}
+          ttlSecondsAfterFinished: {{ env "KLYSHKO_TTL_SECONDS_AFTER_FINISHED" | default 120 }}
+          generator:
+            image: {{ env "KLYSHKO_GENERATOR_IMAGE" | default "ghcr.io/carbynestack/klyshko-mp-spdz:latest" }}
```

Comment:
tags should be pinned to exact versions.
Has tag to be updated once /carbynestack/klyshko#1 has been merged?

Reply:
Set to 0.1.4.